concordium-std/src/traits.rs

```diff
@@ -220,3 +220,26 @@ pub trait ExpectNoneReport {
     /// [fail](macro.fail.html) with the given message, instead of `panic`.
     fn expect_none_report(self, msg: &str);
 }
+
+/// A wrapper trait for 'Serial' with additional contextual information, namely
```

Comment:
What does a "wrapper trait" mean?

Additionally, you could explain the "size_length property" in more understandable terms, as, e.g., the number of bytes to use to record the length of the data.